index.html

```diff
@@ -0,0 +1,611 @@
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/grey-star.svg" alt="negative star rating" title="Negative Star Rating">
+                  <p>$5.00</p>
+                </figcaption>
+              </figure>
+            </li>
+            <li class="app-store-item-preview swiper-slide">
+              <figure>
+                <img src="/img/app-store-placeholder.png" alt="App Store Placeholder Image" title="App Store Placeholder Image">
+                <p class="review-text-overlay">Lorem ipsum dolor sit amet consectetur adipisicing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam quis etu nostrud exercitation ullamco laboris</p>
+                <figcaption>
+                  <h2>Mobile Coach</h2>
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/grey-star.svg" alt="negative star rating" title="Negative Star Rating">
+                  <p><span>$5.00</span> Free!</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <div class="swiper-pagination swiper-pagination-2"></div>
+        </div>
+      </section>
+      <section data-section="download-app" class="l-flex-container download-left">
+        <img src="/img/phones.png" alt="Image of iPhones showing AppBox logo" title="AppBox iPhone" width="439" height="605">
+        <div class="download-text">
+          <h1><strong>Download</strong> Our App</h1>
+          <p class="download-text">Choose your native platform and download the app <em>free!</em></p>
+          <ul>
+            <li><a class="btn-large" target="_blank" href="https://play.google.com/store/apps?hl=en"><span class="svg-google"></span>Google Play</a></li>
+            <li><a class="btn-large" target="_blank" href="https://itunes.apple.com/us/genre/ios/id36?mt=8&cid=wwa-us-kwg-features"><span class="icon-apple2"></span>iOS App Store</a></li>
+          </ul>
+          <aside class="newsletter">
+            <form>
+              <label for="subscribe">Subscribe To Newsletter</label>
+              <input id="subscribe" type="email" name="email" placeholder="Enter your email address">
+              <button class="btn-small" type="submit">Submit</button>
```

Comment:
This button is not sitting in line with the input on Chrome. Expect to hear from the designer.